modules/distr-tracing-tempo-install-web-console.adoc

```diff
@@ -0,0 +1,146 @@
+// Module included in the following assemblies:
+//
+//* distr_tracing_tempo/distr-tracing-tempo-installing.adoc
+
+:_content-type: PROCEDURE
+[id="distr-tracing-tempo-install-web-console_{context}"]
+= Installing the {TempoShortName} by using the OpenShift web console
+
+You can install the {TempoShortName} in the OpenShift web console.
+
+.Prerequisites
+
+* You are logged in to the OpenShift xref:https://docs.openshift.com/container-platform/latest/web_console/web-console.html[web console] as a xref:https://docs.openshift.com/container-platform/latest/admin_solutions/user_role_mgmt.html#creating-a-cluster-administrator[cluster administrator with the `cluster-admin` role].
+// https://docs.openshift.com/container-platform/latest/web_console/web-console.html
+// https://docs.openshift.com/container-platform/latest/admin_solutions/user_role_mgmt.html#creating-a-cluster-administrator
+
+* You are using a supported provider of object storage: link:https://www.redhat.com/en/technologies/cloud-computing/openshift-data-foundation[Red Hat OpenShift Data Foundation], link:https://min.io/[MinIO], link:https://aws.amazon.com/s3/[Amazon S3], link:https://azure.microsoft.com/en-us/products/storage/blobs/[Azure Blob Storage], link:https://cloud.google.com/storage/[Google Cloud Storage].
+
+.Procedure
+
+. Install the {TempoOperator}:
+
+.. In the *Administrator* view of the OpenShift web console, go to *Operators* -> *OperatorHub* and search for `{TempoOperator}`.
+
+.. Select *{TempoOperator}* that is *OpenShift Operator for Tempo* -> *Install* -> *Install* -> *View Operator*.
++
+[IMPORTANT]
+====
+This installs the Operator with the default presets:
+
+* *Update channel* -> *stable*
+* *Installation mode* -> *All namespaces on the cluster*
+* *Installed Namespace* -> *openshift-operators*
```

Comment:
In the product build, we have default namespace 'openshift-tempo-operator' for Tempo operator installation.